packages/forms-web-app/src/controllers/lpa-dashboard/appeal-details.js

```diff
@@ -1,35 +1,48 @@
-const { getAppealByLPACodeAndId } = require('../../lib/appeals-api-wrapper');
+const {
+	getAppealByLPACodeAndId,
+	getAppealDocumentMetaData
+} = require('../../lib/appeals-api-wrapper');
 const {
 	VIEW: {
 		LPA_DASHBOARD: { DASHBOARD, APPEAL_DETAILS }
 	}
 } = require('../../lib/views');
-const {
-	constants: { APPEAL_ID }
-} = require('@pins/business-rules');
+
 const dateFns = require('date-fns');
 
+const HAS_APPEAL_TYPE = 'Householder (HAS) Appeal';
```

Comment:
can you use constant in `packages/common/src/constants.js`?

Reply:
No, they are different values to what is coming from the seed data.